### PR TITLE
[Live-Site] Created video screen

### LINKED ISSUE #436: [Live-Site] Create the video screen share component
image
> Component to create
Figma link - https://www.figma.com/file/O8QLwPzTuH3RIWoYdxitln/Live-Page?node-id=0-1&t=Tm6ZyIzHAvbvrMGj-0

### PR DESCRIPTION
**Pull request by satyam73**
### Issue #436 

Closes #436 

### What is the change?

Created video share ui for live-site. Also 

##### Example :

https://user-images.githubusercontent.com/85296770/232246142-28ff8ac2-61ef-4116-8261-3360442893b1.mp4
> Video for reference

### Is Development Tested?

- [x] Yes
- [ ] No

### CODE REVIEW

File: tests/acceptance/live-test.js [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
+  test('video exists!', async function (assert) {
+    assert.expect(5);
+    await visit('/live');
+
+    const controller = this.owner.lookup('controller:live');
+
+    assert.strictEqual(currentURL(), '/live', 'We are on the live page!');
     assert.dom('[data-test-tabs]').exists();
+
+    await new Promise((resolve) => later(resolve, 4000));
```
Comment (bhtibrewal) [inline tests/acceptance/live-test.js:28]: Don't we have some fakeTimers here? it's not recommended to make the test wait

Comment (bhtibrewal) [reply tests/acceptance/live-test.js:28]: okay

**Review by bhtibrewal — APPROVED** [review]
looks good to me

File: app/templates/live.hbs [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
   @activeTab={{this.activeTab}}
   @tabHandler={{this.tabHandler}}
 />
-<p>{{this.activeTab}} Component</p>
\ No newline at end of file
+<div id='live' data-test-live>
+  {{#if (eq this.activeTab 'Screenshare')}}
+    <div class='live__video-container' data-test-video-container>
+      {{#if this.isLoading}}
+        <Reusables::Skeleton />
```
Comment (SanketDhabarde) [inline app/templates/live.hbs:11]: If we are adding a Skeleton component to the video component only, we can add it for tabs as well. Just for consistency

File: app/styles/live.module.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#live {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 16px;
```
Comment (SanketDhabarde) [inline app/styles/live.module.css:5]: Add `font-size` in rem. This is already followed in the codebase, you will find everywhere `font-size` in rems.

**Review by SanketDhabarde — DISMISSED** [review]
Just NIT comment and Suggestion. Also please follow the proper commit message(Refer to https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/). Everything else looks good.

File: app/styles/live.module.css (hunk 3 above)
Comment (satyam73) [reply app/styles/live.module.css]: Yeah👍

File: app/templates/live.hbs (hunk 2 above)
Comment (satyam73) [reply app/templates/live.hbs:12]: Right now I have added it for video screen when will layout everything then we will add in others too.
Is it seems good?

File: tests/acceptance/live-test.js (hunk 1 above)
Comment (satyam73) [reply tests/acceptance/live-test.js]: Hey @bhtibrewal 
I have changed the approach for tests by using waitFor helper in tests

File: app/templates/live.hbs (hunk 2 above)
Comment (SanketDhabarde) [reply app/templates/live.hbs:12]: Sure np. It's just it doesn't looks good only on video screen

File: app/styles/live.module.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#live {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    /* font-size: 1rem; */
```
Comment (SanketDhabarde) [inline app/styles/live.module.css:6]: Remove comments

File: app/templates/live.hbs (hunk 2 above)
Comment (satyam73) [reply app/templates/live.hbs:12]: Yeah I understand

File: app/styles/skeleton.module.css [hunk 5]
```diff
@@ -0,0 +1,18 @@
+.skeleton {
+    width: 100%;
+    background-color: var(--color-offwhite);
+    height: 100%;
+    animation: pulse 2s infinite reverse;
+    border-radius: 0.5rem;
+}
+
+
```
Comment (rohan09-raj) [inline app/styles/skeleton.module.css:9]: NIT: Extra line can be removed

**Review by rohan09-raj — COMMENTED** [review]
Can we please add TODO comments to remove the dummy files, code, resources, etc ?
So that we do not forget to remove them and keep the project structure clean from unwanted resources/code

File: app/styles/skeleton.module.css (hunk 5 above)
Comment (satyam73) [reply app/styles/skeleton.module.css]: Resolved✅

File: app/templates/live.hbs [hunk 6]
```diff
@@ -1,6 +1,20 @@
...
+    @tabs={{this.TABS}}
+    @activeTab={{this.activeTab}}
+    @tabHandler={{this.tabHandler}}
+  />
+  {{#if (eq this.activeTab 'Screenshare')}}
+    <div class='live__video-container' data-test-video-container>
+      {{#if this.isLoading}}
+        <Reusables::Skeleton />
+      {{else}}
+        <VideoScreen @src='/assets/birds_footage.mp4' />
```
Comment (SanketDhabarde) [inline app/templates/live.hbs:14]: You can add the TODO text here. as the source is dummy as of now

File: app/templates/live.hbs [hunk 7]
```diff
@@ -1,6 +1,20 @@
...
+      {{/if}}
+    </div>
+  {{else}}
+    <p>{{this.activeTab}} Component</p>
```
Comment (SanketDhabarde) [inline app/templates/live.hbs:18]: We can also add TODO here as new components will be added. And this dummy text will be removed.

Comment (satyam73) [reply app/templates/live.hbs:22]: Added

File: app/templates/live.hbs (hunk 6 above)
Comment (satyam73) [reply app/templates/live.hbs:17]: Added

**Review by rohan09-raj — APPROVED** [review]

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**